rpcs3/Emu/RSX/GL/GLGSRender.h

```diff
@@ -18,9 +18,11 @@ struct work_item
 	std::mutex guard_mutex;
 	
 	u32  address_to_flush = 0;
-	bool processed = false;
-	bool result = false;
-	bool received = false;
+	gl::texture_cache::cached_rtt_section *section_to_flush = nullptr;
+
+	volatile bool processed = false;
```

Comment:
As far as I know x86 should not require explicit flushing so as long as the value is fetched from memory on consumer wake it should be fine. The values are just passed as a function result and the consumer does not actually use them.